parm/metplus/GridStat_APCP01h.conf

```diff
@@ -246,13 +246,13 @@ GRID_STAT_OUTPUT_FLAG_NBRCNT = STAT
 
 # NetCDF matched pairs output file
 #GRID_STAT_NC_PAIRS_VAR_NAME =
-GRID_STAT_NC_PAIRS_FLAG_LATLON = TRUE
-GRID_STAT_NC_PAIRS_FLAG_RAW = TRUE
+GRID_STAT_NC_PAIRS_FLAG_LATLON = FALSE
```

Comment:
(see [here](https://met.readthedocs.io/en/latest/Users_Guide/grid-stat.html?highlight=nc_pairs_flag%20latlon)).

Reply:
@gsketefian I discussed this with @michelleharrold and we decided it would be best to have these turned off by default. The ncpairs files are useful for checking that verification fields are created correctly, but the files are not needed in order to create the verification statistics. They can be rather large files as well, so it will help with reducing the footprint of the verification WE2E tests.